src/Swarm/Game/Scenario/Topography/Navigation/Portal.hs

```diff
@@ -72,6 +74,7 @@ data Portal = Portal
   { entrance :: WaypointName
   , exitInfo :: PortalExit
   , consistent :: Bool
+  , reorient :: PlanarRelativeDir
```

Comment:
Do you think there is any need/desire for portals that can do *absolute* (instead of relative) reorientation?  *e.g.* imagine descending into a cave where you start at the end of a long, narrow passage; it might be nice to always start facing down the passage no matter which direction you were facing when you entered the cave.  Just to be clear, I don't think this is something that needs to be addressed in this PR, only some idle thoughts.